databases/digitalocean-pgbouncer/README.md

```diff
@@ -1,8 +1,8 @@
-# PGBouncer on DigitalOcean
+# PgBouncer on DigitalOcean
 
-We want to test DigitalOcean and pgBouncer setup with Prisma client. Prisma client constructor has a `forceTransactions` flag to enable support for pgBouncer.
+We want to test DigitalOcean and PgBouncer setup with Prisma Client. Prisma Query Engine has a `pgbouncer=true` query param to enable support for pgBouncer.
```

Comment:
1. We still need custom configuration. What prompted us to change from client constructor parameter (`forceTransactions`) to a query string parameter? 

2. Are there other parameters that are Prisma specific and not native to the database?

Reply:
> We still need custom configuration

What does this refer to?

> What prompted us to change from client constructor parameter (forceTransactions) to a query string parameter?

See internal discussion: https://prisma-company.slack.com/archives/C1FPU5FPT/p1589560375303200

> Are there other parameters that are Prisma specific and not native to the database?

Yes, the connection limits etc.